plugins/backend-ekg/src/Cardano/BM/Backend/EKGView.lhs

```diff
@@ -148,8 +148,11 @@ ekgTrace ekg _c =
 
         modifyMVar_ (getEV ekgview) $ \ekgup -> do
             let -- strip off some prefixes not necessary for display
-                lognam1 = fromMaybe loname $ stripPrefix "#ekgview.#aggregation." loname
-                logname = fromMaybe lognam1 $ stripPrefix "#ekgview." lognam1
+                stripLPrefix _ [] = Nothing
+                stripLPrefix a (b:bs) | a == b    = Just bs
+                                      | otherwise = Nothing
+                lognam1 = fromMaybe loname $ stripLPrefix "#ekgview" loname
+                logname = fromMaybe lognam1 $ stripLPrefix "#aggregation" lognam1
```

Comment:
I see a lot of places where `loname` is mixed with `logname`. Maybe it would be better to use `lo*` _or_ `log*` in all places? :-)

Reply:
good point. I'll do ...